docs/source/data/resolvers.mdx

```diff
@@ -450,6 +450,33 @@ context: async () => ({
 }
 ```
 
+#### Throwing errors in `context`
+
+By default, if your `context` function throws an error, Apollo Server returns that error to the user in a JSON  response with an HTTP status code of 500. If the thrown error is not a `GraphQLError`, the error's message will be prepended with `"Context creation failed: "`.
```

Comment:
Extra whitespace

Reply:
After the colon in the string? That's intentional and accurate.
I see an extra space after JSON, I can fix that though I don't think it makes a difference when rendered...